pkgs/os-specific/linux/hid-nintendo/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "hid-nintendo";
-  version = "3.1";
+  version = "3.2";
```

Comment:
Technically we don't need to bump this version, but since I'm in here I figured why not?